src/service_provider/service_provider.services.js

```diff
@@ -0,0 +1,30 @@
+import { buildIncludes } from "../utils/includes";
+
+export function FindAllServiceProviders(db, query) {
+    if(query && query.includes){
+        let include = buildIncludes(db, query.includes)
+        return db.service_provider.findAll({ include });
+    }
+    return db.service_provider.findAll();
+}
+
+export function FindAllServiceProvidersById(db, { id }) {
+    return db.service_provider.findOne({ id })
+}
+
+export async function CreateServiceProvider(db, data) {
+    return db.service_provider.create(data)
```

Comment:
Are we expecting the `createdAt` / `updatedAt` to be provided by the user? Or should the argument be a spread `data` with those dates being `Date.now()`?

Reply:
Created at and updated are are auto generated by sequelize

Reply:
Ah ok, it's not explicitly defined like in Knex so I didn't know if that was the case.